projects/router/package.json

```diff
@@ -10,10 +10,10 @@
     "analytics"
   ],
   "peerDependencies": {
-    "@angular/common": "^14.0.0",
-    "@angular/core": "^14.0.0",
-    "@angular/router": "^14.0.0",
-    "@ngx-matomo/tracker": "3.2.0"
+    "@angular/common": "^15.0.0",
+    "@angular/core": "^15.0.0",
+    "@angular/router": "^15.0.0",
+    "@ngx-matomo/tracker": "3.3.0"
```

Comment:
Peer dependency to tracker package will be updated automatically when merged, so it should not be manually changed.